integrations/access/common/recipient.go

```diff
@@ -23,6 +23,16 @@ import (
 	"github.com/gravitational/teleport/integrations/lib/stringset"
 )
 
+const (
+	// RecipientKindSchedule shows a recipient is a schedule.
+	RecipientKindSchedule = "schedule"
+)
+
+const (
+	// ReqAnnotationSchedulesKey is the request annotation key at which schedules are stored for access plugins.
+	ReqAnnotationSchedulesKey = "teleport.dev/schedules"
```

Comment:
Move the label to package `types` where we have all the labels defined. It's easier to reason about the labels we use.

Use the `types.TeleportNamespace` for namespace